EarthAsDMAnalyzer/plugins/EarthAsDMAnalyzer.cc

```diff
@@ -204,6 +207,20 @@ class EarthAsDMAnalyzer : public edm::one::EDAnalyzer<edm::one::SharedResources>
   std::vector<float> muon_dtSeg_eta_;
   std::vector<float> muon_dtSeg_phi_;
 
+  std::vector<float> Track_Reconsruction_Score_;
+  std::vector<float> muon_dtSeg_globY_Hits_rPhi_; //dtGlobalPointYValues
+  std::vector<float> muon_dtSeg_t0timing_Hits_rPhi_; //t0timingValues
+  std::vector<float> muon_dtSeg_globY_Hits_rZ_; //dtGlobalPointYValues_Ztiming
+  std::vector<float> muon_dtSeg_t0timing_rZ_hits_;
+
+  std::vector<float> Track_CorrelationFactor_rPhi_Timing_;
+  std::vector<float> Track_CorrelationFactor_rZ_Timing_;
+
+  std::vector<int> muon_dtSeg_StationHit_;
+  std::vector<int> muon_dtSeg_SectorHit_;
+  std::vector<std::pair<int, int>> muon_dtSeg_stationSectorHit_rPhi_;
```

Comment:
```suggestion
  std::vector<std::pair<int, int>> muon_dtSeg_rPhi_station;
```
* and then why do we need `muon_dtSeg_station_`?